src/index.js

```diff
@@ -14,8 +14,9 @@ server.register(fastifyCors, {});
 
 server.get("/:uname", async (request, reply) => {
   if (!request.params.uname) {
-    reply.code(404).send("Data not available");
-    return;
+    const sql = "SELECT * FROM cv_data";
```

Comment:
This returns way too much, I'd say it should return only the username.
I'll create a new issue to fix this.